mscore/capella.cpp

```diff
@@ -616,7 +616,8 @@ static Fraction readCapVoice(Score* score, CapVoice* cvoice, int staffIdx, const
                         if (tuplet) {
                               if (++nTuplet >= tupletCount) {
                                     tick = tupletTick + tuplet->actualTicks();
-                                    tuplet = 0;
+                                    delete tuplet;
```

Comment:
I don't think this is correct. "tuplet" is a pointer to the newly created tuplet and it is reset to 0 when the tuplet is full so that if a new tuplet is found in the Capella file, a new tuplet in MuseScore score is created (see line 562). But the tuplet to which the pointer was pointing at is part of the score, so I don't think it should be deleted.

Reply:
By the way, Capella tests fail on this PR.

Reply:
Unobvious code, the static analyzer decided that there was a memory leak.
Thanks, I’ll think about how to make the analyzer not think that there is a problem.

Reply:
@AntonioBL I see if the item will be added, depending on the condition
```
if (!o->invisible || voice == 0) {
```
Accordingly, a tuplet may not be added and there will be a memory leak, right?
Maybe you know this situation is typical or exceptional?

Reply:
@AntonioBL @dmitrio95 I ask you to look, is it correct now?

Reply:
I think so. For practically all of the cases, that tuplet will not be deleted (i.e. it will be filled by chord or rest elements).
In principle, by looking at the code, tuplet->elements() could be empty only for tuplets made of invisible rests. This could be a problem only if the rests are voice 0 rests, but it is a remote case, and the original import code is wrong as well.
I may be lazy, but I don't think it is worth the effort to consider also that remote corner case in the code, considering that tuplet import in capella files is limited to simple tuplets only (complex tuplets, especially for old capella files, are encoded in a way which makes it difficult to analyze tuplet duration and its elements a priori, if I remember correctly).
In the years, there were no requests for complex tuplet import.
In summary: for me it is ok. Thank you. 👍